xmtp_mls/src/groups/mod.rs

```diff
@@ -228,6 +228,10 @@ pub struct MlsGroup<C> {
     mutex: Arc<Mutex<()>>,
 }
 
+// C is wrapped in an Arc, and so is the Mutex, so this is fine.
+unsafe impl<C> Send for MlsGroup<C> {}
+unsafe impl<C> Sync for MlsGroup<C> {}
```

Comment:
@insipx This is so we can use this feature on wasm since the Client is neither Send nor Sync there. Everything in MlsGroup that isn't Send/Sync already is wrapped in an Arc, so this should be fine.

Reply:
If this is only for wasm, we should have it under `#[cfg(target_arch = "wasm32")]`